index.html

```diff
@@ -0,0 +1,44 @@
+-+<!DOCTYPE html>
```

Comment:
i think this -+ shouldn't be there......

Reply:
It came by default, well I have remove it.